stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/ChatInfoItem.kt

```diff
@@ -0,0 +1,101 @@
+package io.getstream.chat.ui.sample.feature.chat.info
+
+import androidx.annotation.ColorRes
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import io.getstream.chat.android.client.models.Member
+import io.getstream.chat.ui.sample.R
+
+sealed class ChatInfoItem {
+
+    val id: String
+        get() = when (this) {
+            is MemberItem -> member.getUserId()
+            else -> this::class.java.simpleName
+        }
+
+    data class MemberItem(val member: Member) : ChatInfoItem()
+    object Separator : ChatInfoItem()
+
+    sealed class Option : ChatInfoItem() {
+
+        @get:DrawableRes
+        abstract val iconResId: Int
+
+        @get:StringRes
+        abstract val textResId: Int
+
+        @get:ColorRes
+        abstract val tintResId: Int
+
+        open val showRightArrow: Boolean = true
+
+        object SharedMedia : Option() {
+            override val iconResId: Int
+                get() = R.drawable.ic_media
```

Comment:
Yes, it's because annotations (e.g. `  @get:DrawableRes`) applies only for getters and there is no error when we try to set a different value directly

Reply:
I see, it's because we can only annotate the getter for an abstract property, as it has no field. Makes sense!